lib/helpers/index.js

```diff
@@ -8,6 +10,13 @@ exports.buildQueryRequest = ({ skip, limit, sort, projection, find }) => {
     result.options.skip = skip || 0
     result.options.limit = limit || 100
     if (find) result.query = JSON.parse(find)
+    // URL encoded _id is converted to ObjectId as it will be sent as string
+    if (result.query._id && result.query._id.$in) {
```

Comment:
Maybe instead of commenting what the code is doing since it's a bit complex, can we abstract this to a method which is self explanatory? 
```suggestion
    function processQueryId(query) {
  if (query._id) {
    if (query._id.$in && Array.isArray(query._id.$in)) {
      query._id.$in = query._id.$in.map(convertToObjectId);
    } else if (typeof query._id === 'string') {
      query._id = convertToObjectId(query._id);
    }
  }
  return query;
}

exports.buildQueryRequest = ({ skip = 0, limit = 100, sort, projection, find }) => {
  const result = {
    query: {},
    options: {
      skip,
      limit
    }
  };

  try {
    if (find) result.query = JSON.parse(find);
    result.query = processQueryId(result.query);
    if (sort) result.options.sort = JSON.parse(sort);
    if (projection) result.options.projection = JSON.parse(projection);
  } catch (error) {
    return { error: error.message };
  }

  return result;
};
```